test/metabase/upload_test.clj

```diff
@@ -169,17 +169,23 @@
            [" 2022-01-01 "                    #t "2022-01-01"             date-type]
            [" 2022-02-30 "                    " 2022-02-30 "              vchar-type]
            [" -2022-01-01 "                   #t "-2022-01-01"            date-type]
+           [" Jan 1 2018"                     #t "2018-01-01"             date-type]
```

Comment:
How about adding a test for `Jan 01 2018`?

Reply:
Good point, covered here [f4bc5df](https://github.com/metabase/metabase/pull/38276/commits/f4bc5df1e621c469cda12553e51d9c234998a5ec)